KeenClient/KIOEventStore.m

```diff
@@ -685,6 +689,12 @@ - (void)closeDB {
 }
 
 - (id)convertNSDateToISO8601:(id)date {
+    __block NSString *iso8601 = nil;
```

Comment:
@terryhorner what do you think we should do? The method relies on sqlite3, so it's gonna throw an error if it's called and the database is closed.